internal/wasm/gofunc.go

```diff
@@ -148,8 +148,19 @@ func newModuleVal(m api.Module) reflect.Value {
 	return val
 }
 
+// RequireFunctionType returns the function type corresponding to the function
+// signature or panics if invalid.
+func RequireFunctionType(fn interface{}) *FunctionType {
```

Comment:
Go folks tend to prefix this type of function `Must` (eg regex.MustCompile), and I feel this would belong to testing etc.

Reply:
sgtm